src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -209,10 +209,18 @@ def get_custom_clouds():
     return [c for c in get_clouds() if c.name not in known_cloud_names]
 
 
-def get_clouds():
-    if not os.path.isfile(CLOUD_CONFIG_FILE):
-        for c in KNOWN_CLOUDS:
+def _init_known_clouds():
+    config = configparser.SafeConfigParser()
+    config.read(CLOUD_CONFIG_FILE)
```

Comment:
this file is read and parsed twice (the other one is on line 225). can we avoid it?